src/Form/Extension/Api/ProductTypeExtension.php

```diff
@@ -26,10 +26,15 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
     /**
      * {@inheritdoc}
      */
-    public static function getExtendedTypes()
+    public static function getExtendedTypes(): iterable
     {
         return [
             ProductType::class,
         ];
     }
+
+    public function getExtendedType(): string
+    {
+        return ProductType::class;
+    }
```

Comment:
Do we really need to add that ?

Reply:
not sure, saw in sylius core something similar so I added it, we can drop this later

Reply:
It's deprecated, remove it.